dair_pll/vis_utils.py

```diff
@@ -49,19 +50,37 @@ def generate_visualization_system(
     base system has a copy. Each illustration geometry element in these two
     copies is uniformly colored to be visually distinguishable.
 
+    The copy of the base system can optionally be rendered in its learned
+    geometry.
+
     Args:
         base_system: System to be visualized.
+        learned_system: Optionally, the learned system so the predicted
+          trajectory is rendered with the learned geometry.
         base_system_color: Color to repaint every thing in base system.
         learned_system_color: Color to repaint every thing in duplicated system.
 
     Returns:
         New ``DrakeSystem`` with doubled state and repainted elements.
     """
-    # Creates duplicated system
+    # Start with true base system.
     double_urdfs = deepcopy(base_system.urdfs)
     double_urdfs.update({
-        (k + LEARNED_TAG): v for k, v in base_system.urdfs.items()
+        k: v for k, v in base_system.urdfs.items()
```

Comment:
this appears to be a no-op

Reply:
Good catch -- this and the copy-paste code block below were edited versions of lines in my fork that did do distinct and real tasks (namely find a geometrically accurate version of URDF used for initialization), but the edits to keep that other functionality out of this PR rendered these lines mostly not useful.  I removed this line and combined the others as suggested.